### PR TITLE
tests: better detect rule name

### PR DESCRIPTION
**Pull request by williballenthin**
from https://github.com/mandiant/capa-rules/pull/647

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -69,7 +69,7 @@ jobs:
         cd rules/
         for changed_file in ${{ steps.files.outputs.added_modified }} ${{ steps.files.outputs.renamed }}; do
           if [[ ! $changed_file =~ .git|.md ]]; then
-            tag=$(grep 'name:' $changed_file | sed 's/^.*: //')
+            tag=$(grep '\sname:' $changed_file | sed 's/^.*: //')
```
Comment (williballenthin) [inline .github/workflows/tests.yml:72]: im not sure if this needs to be escaped further. im inclined to test in prod, but what does the reviewer think?

Comment (williballenthin) [reply .github/workflows/tests.yml:72]:
```suggestion
            tag=$(grep '\\sname:' $changed_file | sed 's/^.*: //')
```

not sure

**Review by mr-tz — APPROVED** [review]
good catch

File: .github/workflows/tests.yml (hunk 1 above)
Comment (williballenthin) [reply .github/workflows/tests.yml:72]: i dont think further quoting is needed here

https://yaml-multiline.info/

**Pull request merged**